openedx/api.py

```diff
@@ -886,12 +886,13 @@ def unsubscribe_from_edx_course_emails(user, course_run):
     return result
 
 
-def validate_username_with_edx(username):
+def validate_username_email_with_edx(username, email):
```

Comment:
If there is a validation error on edX we are not returning that for the email at the end of this method. It only returns the msg for the username.